FAQs.md

```diff
@@ -1,113 +1,135 @@
+This is the list of frequently asked questions for Jockey.
+It is intended to serve as a self-service utility document.
+This document may be updated from time to time to suit any new changes to the software as new features/changes are implemented.
+
 
 1. **What is Jockey?**
 
-Jockey is a music player for Android based on Google's Material Design standards. The point of Jockey is to be a simple, lightweight media player with features designed for music enthusiasts and casual listeners alike. 
+Jockey is a music player for Android based on Google's Material Design standards.
+The point of Jockey is to be a simple, lightweight media player with features designed for casual listeners and music enthusiasts alike.
 
 
-2. **What are the features of Jockey?**
+2. **What are its features?**
 
-Jockey is designed with a lot of features that are not common in every other music player. The features are;
+Jockey is designed with a lot of features that aren't present in many other music players. Some examples are:
 
 - Customizable interface with 81 color combinations to choose from
 - Last.fm integration for retrieving artist images and descriptions
-- Smart Playlists with fully customizable rules
+- "Smart" playlists with fully customizable rules
 - Custom equalizers
 - Gapless playback
-- Home screen widgets
-- sleep timer
+- Homescreen widgets
+- Sleep timer
+
+Jockey also contains basic features that should be expected in any half-decent music player. Some examples are:
+
+- Playlist editing
+- Lockscreen controls and artwork support
+- Fluid navigation
 
-Jockey also contains basic features expected in a music player which include;
 
-- playlist editing
-- lockscreen controls and artwork
-- fluid navigation between all of your content
+3. **How do I create a playlist?**
 
+To create a playlist, navigate to the playlist tab by swiping left or right.
+Once there, click the plus (+) sign below and two options will pop up: "Playlist", and "Smart Playlist".
+Click the plus sign beside playlist, input the playlist name, and click on "create".
 
 
-3. **How do I create a create a playlist?**
+4. **What is a "smart playlist"?**
 
-To create a playlist on Jockey, navigate to the playlist tab by swiping left or right. On the playlist, tab click the plus (+) sign below and two options will pop up Smart Playlist and Playlist. Click the plus sign beside playlist, input playlist name and click on create.
+Smart playlist (as the name implies) is a playlist that automatically sorts and adds music to the list according to customized rules / set parameters.
 
-4. **What is a smart playlist?**
 
-Smart playlist as the name implies is a playlist that automatically sorts and adds music to the list according to customized rules or parameters set. 
+5. **Can I import / export playlists?**
 
-5. **Can I import and export playlists on Jockey?**
+No, playlists can't be imported or exported. Apologies.
```

Comment:
I'd recommend trying to frame this a bit more positively. Perhaps say that playlists created on the phone can be used from other applications (and vice versa), but cannot be exported to another device or computer?